internal/progress.go

```diff
@@ -16,26 +16,35 @@
 // HarbourBridge.
 package internal
 
-import "fmt"
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
 
 // Progress provides console progress functionality. i.e. it reports what
 // percentage of a task is complete to the console, overwriting previous
 // progress percentage with new progress.
 type Progress struct {
-	total    int64  // How much we have to do.
-	progress int64  // How much we have done so far.
-	pct      int    // Percentage done i.e. progress/total * 100
-	message  string // Name of task being monitored.
-	verbose  bool   // If true, print detailed info about each progress step.
+	total      int64  // How much we have to do.
+	progress   int64  // How much we have done so far.
+	pct        int    // Percentage done i.e. progress/total * 100
+	message    string // Name of task being monitored.
+	verbose    bool   // If true, print detailed info about each progress step.
+	fractional bool   // If true, report progress in fractions instead of percentages.
 }
```

Comment:
I think this change makes in sense in general, but for the case of FKs I wonder if we can also output the absolute number of foreign keys processed vs remaining (potentially also in-flight).

Reply:
Currently, this change displays processed / total for ex: "40/100", should we output 40 processed, 60 remaining instead?

Reply:
What you have right now is perfect, I was mislead by the variable name initially and thought that it's going to print something like 0.1%, 0.2% and so on ...